### ISSUE TITLE
Broken release?

### ISSUE THREAD

**Issue by MrOzean** [matoous/go-nanoid#16]
This dependency don't working
```
require (
	github.com/matoous/go-nanoid v1.4.0
)
```

> go:  github.com/matoous/go-nanoid@v1.4.0: reading github.com/matoous/go-nanoid/go.mod at revision v1.4.0: unknown revision v1.4.0

But working this. 
```
require (
	github.com/matoous/go-nanoid v.1.4.0
)
```
Which change automatically in my go.mod to:
```
require (
	github.com/matoous/go-nanoid v1.3.1-0.20200518070138-513938f8e67d
)
```

**Comment by matoous** [comment]
Was unable to fix the issue, please require v1.4.1 instead.

```
require (
	github.com/matoous/go-nanoid v1.4.1
)
```

**Comment by MrOzean** [comment]
Thanks for fast fix
